data_model/src/permission.rs

```diff
@@ -65,32 +36,34 @@ mod model {
         Serialize,
         IntoSchema,
         Display,
-        Getters,
     )]
     #[ffi_type]
-    #[display(fmt = "PERMISSION `{id}` = `{payload}`")]
-    #[getset(get = "pub")]
+    #[display(fmt = "PERMISSION `{name}` = `{payload}`")]
     pub struct Permission {
         /// Refers to a type defined in [`crate::executor::ExecutorDataModel`].
-        pub id: PermissionId,
+        pub name: Ident,
```

Comment:
Why not keep it a `Name` to prevent reserved characters from being used?

Reply:
It's unclear to me atm which type schema identifiers should have